### PR TITLE
doc: Updates to panel user guide.

### PR DESCRIPTION
**Pull request by alanconway**

### CODE REVIEW

**Comment by PeterYurkovich** [comment]
I'll leave this PR up for further changes and discussions. I'll throw a link in the last one to this one for the links you sent in slack. Let me know when you are ready for a LGTM on this one

**Comment by alanconway** [comment]
@gabriel-rh any initial feedback before we merge this? It's upstream so we can incorporate your feedback later if not ready now.

File: doc/README.adoc [hunk 1]
```diff
@@ -4,20 +4,20 @@
 
 The troubleshooting panel displays a graph of related resources and observability signals.
 Clicking on a node in the graph opens a console page showing the details of each resource or signal.
-Nodes in the graph represent a resource or signal type, edges represent relationships.
+Nodes in the graph represent a type of resource or signal, edges represent relationships.
```
Comment (gabriel-rh) [inline doc/README.adoc:7]:
```suggestion
Nodes in the graph represent a type of resource or signal, while edges represent relationships.
```

File: doc/README.adoc [hunk 2]
```diff
@@ -4,20 +4,20 @@
...
-Lets consider the example of troubleshooting an Alert on the OCP console.
+Consider an example of troubleshooting an Alert on the OCP console.
 
 == Opening the panel
 
 First open the alert of interest in the console.
-Now open the troubleshooting panel [FIXME global button]
+Now open the troubleshooting panel [FIXME screenshot of global button ]
 
 Opening panel shows a _neighbourhood_ of the resource currently displayed in the console.
```
Comment (gabriel-rh) [inline doc/README.adoc:19]:
```suggestion
Opening the panel shows a _neighbourhood_ of the resource currently displayed in the console.
```

File: doc/README.adoc [hunk 3]
```diff
@@ -4,20 +4,20 @@
...
-A neighbourhood graph starts at the current resource, and includes related objects up to
+A neighbourhood is a graph that starts at the current resource, and includes related objects up to
 3 steps away from the starting point.
 
 NOTE: Not all resource types are currently supported, more will be added in future.
```
Comment (gabriel-rh) [inline doc/README.adoc:23]:
```suggestion
NOTE: Not all resource types are currently supported, and more will be added in future.
```

File: doc/README.adoc [hunk 4]
```diff
@@ -27,41 +27,49 @@ For an unsupported resource, the panel will be empty.
...
-<6> Network(6) shows that there are network events, which means the pod has communicated over the network. The remaining nodes are the Service, Deployment and DaemonSet that the pod has communicated with.
-<7> Focus: Clicking on nodes in the graph will change what is shown in the main console page, but the graph will not change.
-    You can also navigate using links on the console page while the panel is open.
-    To update the graph, click "Focus". This will draw a new graph using the resource shown in the main page as the starting point.
-<8> Show Query enables some experimental features, see below.
-
-NOTE: Clicking on a node may sometimes show fewer results than are indicated on the graph. This is a known issue that will be addressed in future.
+<1> Alert(1): This node represents the starting point, a `KubeContainerWaiting` alert that was displayed in the console.
+<2> Pod(1): This node indicates there is a single Pod resource associated with this alert. Clicking on this node will show the pod details in the console.
+<3> Event(2): There are two kuberenetes events associated with the Pod, you can see them by clicking this node.
```
Comment (gabriel-rh) [inline doc/README.adoc:32]:
```suggestion
<3> Event(2): There are two kuberenetes events associated with the Pod, and you can see them by clicking this node.
```

File: doc/README.adoc [hunk 5]
```diff
@@ -27,41 +27,49 @@ For an unsupported resource, the panel will be empty.
...
+<3> Logs(74): The pod has emitted 74 lines of logs, click to show them.
```
Comment (gabriel-rh) [inline doc/README.adoc:33]:
```suggestion
<3> Logs(74): The pod has emitted 74 lines of logs. Click to show them.
```

File: doc/README.adoc [hunk 6]
```diff
@@ -27,41 +27,49 @@ For an unsupported resource, the panel will be empty.
...
+<4> Metrics(105): There are always many metrics associated with every Pod.
+<6> Network(6): There are network events associated with the pod, which means it has communicated with other resources in the cluster.
+    The remaining Service, Deployment and DaemonSet nodes are the resources that the pod has communicated with.
+<7> Focus: Clicking nodes changes what is shown in the main console. You can navigate using links on the console page while the panel is open.
+    The graph will not change until you click "Focus".
+    This will draw a new graph starting from the resource shown in the main console.
+<8> Show Query: enables experimental features, see below.
```
Comment (gabriel-rh) [inline doc/README.adoc:40]:
```suggestion
<8> Show Query: enables experimental features detailed below.
```

File: doc/README.adoc [hunk 7]
```diff
@@ -27,41 +27,49 @@ For an unsupported resource, the panel will be empty.
...
 
 [.border]
-image::images/query-details.png[]
+_image::images/query-details.png[]
 
 <1> Hide Query hides the experimental features.
-<2> The "Korrel8r query" that identifies the starting point for the graph. This query language is experimental and will change in future.
-    It is updated by the "Focus" button to correspond to the resources in the main console window. (TODO upstream reference or no?)
+<2> The query that identifies the starting point for the graph. The format of this query is experimental and may change in future.
+    footnote:[This query language is part of https://korrel8r.github.io/korrel8r:[Korrel8r], the correlation engine used to create the graphs]
```
Comment (gabriel-rh) [inline doc/README.adoc:52]: https://korrel8r.github.io/korrel8r:    -> https://korrel8r.github.io/korrel8r or https://korrel8r.github.io/korrel8r/

**Review by gabriel-rh — COMMENTED** [review]
LGTM

Only one required change - removing : at the end of URL. All others are just suggestions

File: doc/README.adoc (hunk 7 above)
Comment (alanconway) [reply doc/README.adoc:52]: Doh! Colon after URL, shame on me.
All good changes, am afk now but will apply ASAP.

**Comment by alanconway** [comment]
Incorporated changes, @gabriel-rh or @PeterYurkovich, can I get an /approved and and /lgtm?

**Pull request merged**